WebRoot/css/zm.css

```diff
@@ -3574,20 +3574,58 @@ HTML>BODY .appt-selected .appt_allday_body
 .ZmContactSearch {
 	width:500px;
```

Comment:
If this style is related to the component that you have worked on here, how about moving it to properties file?